examples/rbfe_single.py

```diff
@@ -124,25 +117,24 @@ def wrap_method(args: Iterable[Any], fxn: callable):
         cmd_args.num_prod_steps
     )
 
-    vg_fn = jax.value_and_grad(binding_model.loss, argnums=0)
+    vg_fn = jax.value_and_grad(binding_model.loss, argnums=0, has_aux=True)
```

Comment:
nice the `has_aux=True` works now?

Reply:
ah I see - this is done through outside of the custom_vjp via the callback trick.

Reply:
Good catch -- that was in error! Reverted in https://github.com/proteneer/timemachine/pull/360/commits/ca31e56995de24c34fcfe5eaa6f086824857df50